File: java/fibonacci.java
class fibonacci {
    public static void main (String[] args) {
        int n = 10;
        int t1 = 0;
        int t2 = 1;
        int sum = 0;
        System.out.println("Upto "+n+" : ");
        while (sum <= n) {
            System.out.println(sum);
            sum = t1+t2;
            t1 = t2;
            t2 = sum;
        }
    }
}

File: PHP/fibonacci.php
<?php

$limit = $_POST['input'];
$first = 0;
$second = 1;

echo "Fibonacci series upto $limit: <br>";
while ($second <= $limit) {
    echo "$first<br>";
    if ($second <= $limit)
        echo "$second<br>";
    $first += $second;
    $second += $first;
}
